ci/actions-templates/linux-builds-template.yaml

```diff
@@ -46,6 +46,7 @@ jobs:
           - armv7-linux-androideabi       # skip-pr skip-master
           - i686-linux-android            # skip-pr skip-master
           - x86_64-linux-android          # skip-pr skip-master
+          - riscv64gc-unknown-linux-gnu   # skip-pr skip-master
```

Comment:
This may also need a `skip-stable` because we're not going to release this for now.

Reply:
+1 see 67c272166fa3ea1414103f77e4dfd891c58e4580